tests/core/pyspec/eth2spec/test/context.py

```diff
@@ -545,6 +549,7 @@ def wrapper(*args, spec: Spec, **kw):
 with_deneb_and_later = with_all_phases_from(DENEB)
 with_eip6110_and_later = with_all_phases_from(EIP6110)
 with_eip7002_and_later = with_all_phases_from(EIP7002)
+with_whisk_and_later = with_all_phases_from(WHISK)
```

Comment:
Oops, so `with_all_phases_from` only iterates `ALL_PHASES`, which `WHISK` is not part of it now. Hence, Whisk state test cases are still skipped even with `--fork=whisk`. 😬
https://github.com/ethereum/consensus-specs/blob/5c194bb4934917b4bd65e6fb0ad3ce5e459e36af/tests/core/pyspec/eth2spec/test/context.py#L421-L427

A quick solution is adding a new argument `all_phases` to this helper:

```python
def with_all_phases_from(earliest_phase, all_phases=ALL_PHASES):
    """
    A decorator factory for running a tests with every phase except the ones listed
    """
    def decorator(fn):
        return with_phases([phase for phase in all_phases if is_post_fork(phase, earliest_phase)])(fn)
    return decorator
```

and re-write `with_whisk_and_later`:
```python
with_whisk_and_later = with_all_phases_from(WHISK, all_phases=ALLOWED_TEST_RUNNER_FORKS)
```